R/find_regimes.R

```diff
@@ -33,8 +33,14 @@ find_regimes <- function(y,
       chains = chains, ...
     )
     looic <- loo.bayesdfa(fit)
-    loo_bad <- loo::pareto_k_table(looic)["(0.7, 1]", "Count"]
-    loo_very_bad <- loo::pareto_k_table(looic)["(1, Inf)", "Count"]
+    k_table <- loo::pareto_k_table(looic)
+    if (nrow(k_table) == 4) {
+      loo_bad <- k_table["(0.7, 1]", "Count"]
+      loo_very_bad <- k_table["(1, Inf)", "Count"]
+    } else {
+      loo_bad <- k_table[2, "Count"]
+      loo_very_bad <- k_table[3, "Count"]
+    }
```

Comment:
This code should work with any version of the loo package now

Reply:
Once we release the next loo you could change checking the number of rows of the table to checking the version of loo, but either way should work